pyquil/operator_estimation.py

```diff
@@ -768,27 +798,48 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
         for qubit, op_str in max_weight_out_op:
             total_prog += _local_pauli_eig_meas(op_str, qubit)
 
-        # 2. Run the experiment
-        bitstrings = qc.run_and_measure(total_prog, n_shots)
+        if symmetrize:
+            # 1.4 Symmetrize -- flip qubits pre-measurement
+            qubits = max_weight_out_op.get_qubits()
+            ops_strings = _ops_strs_symmetrize(qubits)
+            list_bitstrings_symm = []
+            for ops_str in ops_strings:
```

Comment:
the whole `ops_str` business seems pretty roundabout. Some things I would consider (off the top of my head / haven't thought too deeply about any of these suggestions):

 - increase readability be reducing code indirection. Instead of assigning the return value of `_ops_strs_symmetrize` to a variable which is only used once (?), just stick it in the for loop. Then the helper function can be made into a generator
 - do we need to factor out `_ops_strs_symmetrize`? it's just a call to `itertools`.
 - switch the type of `ops_str` to a numpy array of `0/1` or dtype=bool or a list of ints. This will remove one conversion str->numpy to do the post-quantum flip-back and will not complicate the `ops_str`->program conversion. In the current design, you should be worried that `ops_str` as a string of characters is *never used as a string of characters*. It is always converted to some other piece of data before use. See if you can remove `_ops_str_to_flips` helper function

Reply:
Removed `_ops_strs_symmetrize` as well as `_ops_str_to_flips`